apps/meteor/tests/end-to-end/api/livechat/04-dashboards.ts

```diff
@@ -777,6 +778,198 @@ describe('LIVECHAT - dashboards', function () {
 		});
 	});
 
+	describe('[livechat/analytics/agent-overview] - Average first response time', () => {
+		let agent: { credentials: Credentials; user: IUser & { username: string } };
+		let originalFirstResponseTimeInSeconds: number;
+		let roomId: string;
+		const firstDelayInS = 4;
```

Comment:
Just to follow the same pattern
```suggestion
		const firstDelayInSeconds = 4;
```